node/_http_client.js

```diff
@@ -0,0 +1,897 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
```

Comment:
I suspect this file was accidentally resurrected during a rebase?

Reply:
Oh, thanks! This was a mistake during resolving merge conflicts!